Code2Gether-Discord-Bot.Tests/TestConfig.cs

```diff
@@ -0,0 +1,215 @@
+﻿using Code2Gether_Discord_Bot.Library.BusinessLogic;
+using Code2Gether_Discord_Bot.Library.Models.Repositories.ProjectRepository;
+using Code2Gether_Discord_Bot.Static;
+using Code2Gether_Discord_Bot.Tests.Fakes;
+
+namespace Code2Gether_Discord_Bot.Tests
+{
+    /// <summary>
+    /// This class intantiates all fake objects used inside in unit tests.
+    /// </summary>
+    internal static class TestConfig
+    {
+        #region BusinessLogicFactory
+        /// <summary>
+        /// Instantiates a generic <see cref="Library.BusinessLogic.ExcuseGeneratorLogic"/>.
+        /// </summary>
+        /// <returns>ExcuseGeneratorLogic with irrelevant properties.</returns>
+        public static IBusinessLogic ExcuseGeneratorLogic()
+        {
+            var classContext = typeof(object);
+            var context = CommandContext();
+
+            return BusinessLogicFactory.ExcuseGeneratorLogic(classContext, context);
+        }
+        /// <summary>
+        /// Instantiates a generic <see cref="Library.BusinessLogic.InfoLogic"/>.
+        /// </summary>
+        /// <returns>InfoLogic with irrelevant properties.</returns>
+        public static IBusinessLogic InfoLogic()
+        {
+            var classContext = typeof(object);
+            var context = CommandContext();
+
+            return BusinessLogicFactory.GetInfoLogic(classContext, context);
```

Comment:
Factory should not be used, as it is the production implementation. During testing `new InfoLogic` can be used.

Reply:
Thanks for your review!

Reply:
Fixed.